src/encoded/static/components/browse/PublicationSearchView.js

```diff
@@ -0,0 +1,278 @@
+        }
+    },
+    "date_published" : {
+        "widthMap" : { "sm" : 60, "md" : 60, "lg" : 60 },
+        "render" : function(result, columnDefinition, props, termTransformFxn){
+            const { date_published = null } = result;
+            if (!date_published) return null;
+            return (
+                <span className="value">
+                    <LocalizedTime timestamp={date_published} formatType="date-year" />
+                </span>
+            );
+        }
+    },
+    "journal" : {
+        "widthMap" : { "sm" : 100, "md" : 120, "lg" : 150 },
+        "render" : function(result, columnDefinition, props, termTransformFxn){
+            const { journal = null } = result;
+            if (!journal) return null;
+            return (
+                <span className="value">
+                    <JournalTitle journal={journal} width={props.width} />
+                </span>
+            );
+        }
+    },
+    "number_of_experiment_sets" : {
+        "colTitle" : "Exp Sets",
+        "widthMap" : { "sm" : 50, "md" : 70, "lg" : 80 },
+        "render" : function(result, columnDefinition, props, termTransformFxn){
+            const { number_of_experiment_sets: numSets = null } = result;
+            if (numSets === null) return null;
+            return (
+                <span className="value">
+                    { valueTransforms.decorateNumberWithCommas(numSets) }
+                </span>
+            );
+        }
+    }
+});
+
+/**
+ * This is memoized into a couple of different functions - 1 per column (common colWidth between all cells in same col)
+ * Make sure to keep this in sync with CSS rules (line height, etc.)
+ */
+function maxCharsToShowPerLine(colWidth){
+    const charWidth = 6; // Rough upper estimate (excl "i", "l", ".", etc.) for this font/size/weight
+    // -16px for table cell padding (8px left + right)
+    const textAreaWidth = colWidth - 60 - 16; // Account for line splitting on spaces (not in middle of words)
+    return Math.floor(textAreaWidth / charWidth);
+}
+
+
+class PublicationSearchResultTitle extends React.PureComponent {
+
+    static maxCharsToShowPerLine = memoize(maxCharsToShowPerLine);
+
+    static defaultProps = {
+        "showAbstractBlock" : false
+    };
+
+    constructor(props){
+        super(props);
+        this.onClickTrack = this.onClickTrack.bind(this);
+    }
+
+    onClickTrack(evt){
+        const { result, href, navigate = spcNavigate } = this.props;
+        evt.preventDefault();
+        evt.stopPropagation();
+        analytics.productClick(result, {
+            'list'      : analytics.hrefToListName(href),
+            'position'  : rowNumber + 1
+        }, function(){
+            navigate(object.itemUtil.atId(result));
+        });
+        return false;
+    }
+
+    render(){
+        const { result, detailOpen, toggleDetailOpen, rowNumber, href, navigate = spcNavigate, width: colWidth, showAbstractBlock } = this.props;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['navigate' is assigned a value but never used. (no-unused-vars)](https://app.codacy.com/app/4dn/fourfront/pullRequest?prid=4551745)